docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Forgot your password or username
+
+Via [{{gui}}](https://{{gui_domain}}) you can reset lost password or reclaim your username.
```

Comment:
Suggestion: "Via the [{{gui}}](https://{{gui_domain}}), you can reset a lost password or reclaim your username."